### DISCUSSION TITLE
module 'ifcopenshell.api.project' has no attribute 'create_file'

### DISCUSSION THREAD

**Discussion by fbosche** [IfcOpenShell/IfcOpenShell#5828] [Q&A] [answered]
Hello,

I've been trying to create an IFC file using ifcopenshell-python on a Jupyter Notebook. I have 0.8.0 installed. The code I try is below and comes from the tutorial. Yet, I get stuck at the first hurdle with create_file(). I've looked online but cannot find any hint aside from the ifcopenshell version, which is surely not an issue here. Any idea?

```
import ifcopenshell.api.root
import ifcopenshell.api.unit
import ifcopenshell.api.context
import ifcopenshell.api.project
import ifcopenshell.api.spatial
import ifcopenshell.api.geometry
import ifcopenshell.api.aggregate

# Create a blank model
model = ifcopenshell.api.project.create_file()

```

**Comment by RickBrice** [comment, marked as answer]
Try

~~~
model=ifcopenshell.file(schema=“IFC4X3_ADD2”)
~~~

here is an example for alignments 

https://github.com/civilx64/dev-ifcopenshell-4x3-infra/blob/90607ded1a07a6582009626129f8de4475861454/examples/FHWA_Bridge_Geometry_Manual/Alignment.py

**Reply by fbosche** [reply]
1) Thanks for the super-quick reply.
2) So, is the IfcOpenShell python tutorial just not valid anymore?

**Reply by RickBrice** [reply]
I’m sure it’s valid. Just sharing what works for me

**Reply by fbosche** [reply]
Thanks. This has been very useful. 
But, hopefully, I can get some further insight on this. I'm curious
